relay-server/src/utils/unreal.rs

```diff
@@ -143,7 +143,7 @@ fn merge_unreal_context(event: &mut Event, context: Unreal4Context) {
         event
             .logentry
             .get_or_insert_with(LogEntry::default)
-            .formatted = Annotated::new(msg);
+            .formatted = Annotated::new(msg.into());
```

Comment:
_nit_: Since this is not a test, can we make this more explicit (if `Message` sticks)?

```suggestion
            .formatted = Annotated::new(Message::from(msg));
```

Reply:
Done, and this made notice the type was improperly exported